src/tests/actions/userActions.test.js

```diff
@@ -89,4 +94,83 @@ describe('userAction', () => {
       },
     ]);
   });
+  it('should have social login initiated with google', async () => {
+    const user_data = {
+      user: {
+        access_token: 'token',
+      },
+    };
+    const type = [{ type: SOCIAL_LOGIN_INITIATED }];
+    googleLoginUser('/api/auth/google/', user_data)(store.dispatch);
+    await flushAllPromises();
+    expect(store.getActions()).toEqual(type);
+  });
+  it('should have social login initiated with facebook', async () => {
+    const user_data = {
+      user: {
+        access_token: 'token',
+      },
+    };
+    const type = [{ type: SOCIAL_LOGIN_INITIATED }];
+    facebookLoginUser('/api/auth/facebook/', user_data)(store.dispatch);
+    await flushAllPromises();
+    expect(store.getActions()).toEqual(type);
+  });
+  it('should have social login initiated with google', async () => {
```

Comment:
Lines `97` and `119` have the same string description, please consider changing them to make tests more descriptive